backend/src/core/models/user/user-features.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Column,
+  DataType,
+  Model,
+  Table,
+  ForeignKey,
+  Default,
+} from 'sequelize-typescript'
+import { User } from './user'
+
+@Table({ tableName: 'user_features', underscored: true, timestamps: true })
+export class UserFeatures extends Model<UserFeatures> {
+  @ForeignKey(() => User)
+  @Column({
+    type: DataType.INTEGER,
+    allowNull: false,
+    primaryKey: true,
+  })
+  userId!: number
+
+  @Default(null)
+  @Column({
+    type: DataType.INTEGER,
```

Comment:
string?  the announcementVersion should probably follow the package version (https://semver.org/)

Reply:
Done~